packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -415,11 +384,38 @@ class ScrollingFlameChartRowState<V> extends State<ScrollingFlameChartRow>
   void initState() {
     super.initState();
     scrollController = widget.linkedScrollControllerGroup.addAndGet();
+    extentDelegate = _ScrollingFlameChartRowExtentDelegate(
+      nodeRects: nodes.toPaddedZoomedRects(
+        zoom: widget.zoom,
+        chartStartInset: widget.startInset,
+        chartWidth: widget.width,
+      ),
+      zoom: widget.zoom,
+      chartStartInset: widget.startInset,
+      chartWidth: widget.width,
+    );
   }
 
   @override
   void didUpdateWidget(ScrollingFlameChartRow oldWidget) {
     super.didUpdateWidget(oldWidget);
+    // Avoid calling `extentDelegate.recompute()` when only the selected node
```

Comment:
nit: maybe remove this comment. it is a bit confusing and it is fairly standard to only call update methods based on certain data in the class changing.
If you keep the comment, rewrite it to be a positive statement.